exercises/bowling/src/test/java/BowlingTest.java

```diff
@@ -0,0 +1,268 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class BowlingTest {
+    private BowlingGame game;
+    
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Before
+    public void setup() {
+        game = new BowlingGame();
+    }
+    
+    private void PlayGame(int[] rolls)
+    {
+        for(int pins : rolls)
+        {
+            game.roll(pins);
+        }
+    }
+    
+    @Test
+    public void shouldBeAbleToScoreAGameWithAllZeros() {
+        int[] rolls = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+        PlayGame(rolls);
+        assertEquals(0, game.score());
+    } 
+    
+    @Ignore("Remove to run test")
+    @Test
+    public void shouldBeAbleToScoreAGameWithNoStrikesOrSpares() {
+        int[] rolls = { 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6 };
+        PlayGame(rolls);
+        assertEquals(90, game.score());
+    }
+    
+    @Ignore("Remove to run test")
+    @Test
+    public void aSpareFollowedByZerosIsWorthTenPoints() {
+        int[] rolls = { 6, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+        PlayGame(rolls);
+        assertEquals(10, game.score());
```

Comment:
It might improve readability a little bit if the expected score was a variable instead